src/Notifications/NotificationCenterController.cs

```diff
@@ -48,7 +49,7 @@ public void UpdateNotificationWindowSize(int height)
         }
     }
 
-    public class NotificationCenterController
+    public class NotificationCenterController : IDisposable
```

Comment:
made this class disposable so we can clean up when the parent view extension is Disposed.